client/app/dedicated/server/statistics/dedicated-server-statistics.controller.js

```diff
@@ -67,18 +67,36 @@ angular.module('controllers').controller('controllers.Server.Stats', (
 
   $scope.$on('dedicated.informations.bandwidth', $scope.loadBandwidthInformations);
 
+  function convertData(list) {
+    return _.map(list, value => (value.unit === 'bps' ? (value.y / 1024).toFixed(2) : value.y));
+  }
+
   function createChart(data) {
     $scope.series = [];
     $scope.data = [];
-
-    $scope.labels = _.map(_.get(data, 'download.values'), (value, index) => moment(data.download.pointStart)
-      .add(data.download.pointInterval.millis * index, 'milliseconds')
-      .calendar());
-
+    $scope.labels = _.map(_.get(data, 'download.values'), value => moment.unix(value.timestamp).calendar());
     $scope.series.push($translate.instant('server_tab_STATS_legend_download'));
     $scope.series.push($translate.instant('server_tab_STATS_legend_upload'));
-    $scope.data.push(_.map(_.get(data, 'download.values'), value => value.y));
-    $scope.data.push(_.map(_.get(data, 'upload.values'), value => value.y));
+    $scope.data.push(convertData(_.get(data, 'download.values')));
+    $scope.data.push(convertData(_.get(data, 'upload.values')));
+    const { unit } = _.head(_.get(data, 'download.values'));
```

Comment:
Prefer `first` in case of Lodash bump to v4.17.11 :
```suggestion
    const { unit } = _.first(_.get(data, 'download.values'));
```

Reply:
Haha, I should read the v4.17.11 documentation too